packages/components/src/DataList/components/DataListActions/DataListActions.tsx

```diff
@@ -25,7 +24,7 @@ export function DataListActions<T extends DataListObject>({
   childrenArray.splice(0, exposedActions.length);
 
   return (
-    <DataListOverflowFade>
+    <>
```

Comment:
This impacts both the "row" actions and the "bulk" actions, but has no negative impact on either instance:

<img width="214" alt="image" src="https://github.com/user-attachments/assets/4a7067ba-8abd-4ce3-a529-3291adcee78f">

<img width="321" alt="image" src="https://github.com/user-attachments/assets/0949e43f-b612-4fa3-a725-9cef2585da16">
<img width="341" alt="image" src="https://github.com/user-attachments/assets/68d73756-eb1f-4d19-b790-2f5bbfa320b0">

Reply:
At one point the design direction was to have the bulk actions be scrollable, which is likely why this was implemented, but you can see that they collapse down to the ••• menu once the viewport is narrowed so the gradient fade support added by `DataListOverFlowFade` was not giving us anything, and was in turn:

- coupling the spacing of these actions to the spacing in the filters bar
- clipping the focus shadow of the actions

So two birds with one stone, as they say